test/e2eTests/pages/components/headerComponent.js

```diff
@@ -25,7 +25,8 @@ class HeaderComponent extends Component {
     this.I.say(`${LOG_TAG} Clicking on the home button`);
     this.I.click(this.homeLink);
 
-    this.I.wait(1);
+    this.I.waitForVisible("#home-link");
```

Comment:
Yes, I think this was to make it obvious what selector was being used to wait. I can see where you might be arguing for DRY here, but it will take two searches to find anything using or waiting on homeLink (when looking for \#home-link, one to find this.homeLink is the var name and one to find the uses of this.homeLink... I'll change it over to match your use of the var for consistency.